CPP/Applications/Queue/SimulationNetworkPackets/Simulate.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+using namespace std;
+#include <string>
+#include "Simulate.h"
+
+
+int GetMaxSize(int S[])
```

Comment:
Don't use `int` and this kind of data types -_- use std types such as `uint32_t`